src/go/configgenerator/listener_generator.go

```diff
@@ -601,11 +633,21 @@ func makeTranscoderFilter(serviceInfo *sc.ServiceInfo) *hcmpb.HttpFilter {
 				DescriptorSet: &transcoderpb.GrpcJsonTranscoder_ProtoDescriptorBin{
 					ProtoDescriptorBin: configContent,
 				},
-				IgnoredQueryParameters: []string{"api_key", "key", "access_token"},
-				AutoMapping:            true,
-				ConvertGrpcStatus:      true,
+				AutoMapping:       true,
+				ConvertGrpcStatus: true,
+			}
+			for apiKeyQueryParam := range serviceInfo.ApiKeyQueryParams {
```

Comment:
I would also ensure the value is set to `true`. If if it will never be `false`, it will catch future bugs.

Reply:
Good point.